### PR TITLE
Ballot Robert

### PR DESCRIPTION
**Pull request by JeromeMartinez**
Address some issues from [Robert Wilton's ballot](https://mailarchive.ietf.org/arch/msg/cellar/v65eMDmiELzutxYyFhPXf06mUUQ/).
I don't think that I will handle other comments as I don't think we could do better, I answer to Robert about that.

### CODE REVIEW

**Comment by JeromeMartinez** [comment]
PR rebased after merge of other PRs.

File: ffv1.md [hunk 1]
```diff
@@ -351,28 +354,20 @@ JPEG2000-RCT is a Reversible Color Transform that codes RGB (red, green, blue) P
 SVGI:!---
 SVGI:![svg](rgb1.svg "rgb 1")
 SVGI:!---
-SVGC:rgb1.svg=$$\\\\begin{array}{ccccccc}Cb & = & b - g \\\\\\ Cr & = & r - g \\\\\\ Y & = & g + ( Cb + Cr)>>2 \\\\\\ g & = & Y - ( Cb + Cr ) >> 2 \\\\\\ r & = & Cr + g \\\\\\ b & = & Cb + g \\\\end{array}$$
+SVGC:rgb11.svg=$$\\\\begin{array}{ccccccc}Cb & = & b - g \\\\\\ Cr & = & r - g \\\\\\ Y & = & g + ( Cb + Cr)>>2 \\\\end{array}$$
```
Comment (dericed) [inline ffv1.md:357]: The name of the svg must match in the SVGI and SVGC section, ie match the filename of the svg in lines 355 and 357 as well as 364 and 366 as well as 397 and 399 as well as 406 and 408.

File: ffv1.md [hunk 2]
```diff
@@ -596,6 +617,8 @@ AART:zero_state_(i) = 256 - one_state_(256-i)
 
 #### default\_state\_transition
 
+By default the following State Transition Table is used:
```
Comment (dericed) [inline ffv1.md:620]: I suggest a  comma after the word `default`.

**Comment by JeromeMartinez** [comment]
svg name issues fixed, comma added, @dericed please review again.

File: ffv1.md [hunk 3]
```diff
@@ -394,16 +389,40 @@ In JPEG2000-RCT, the coding order would be left to right and then top to bottom,
...
+SVGI:![svg](rgb4.svg "rgb 4")
+SVGI:!---
+SVGC:rgb4.svg=$$\\\\begin{array}{ccccccc}b & = & Y - (Cb + Cr)>>2 \\\\\\ r & = & Cr + b \\\\\\ g & = & Cb + b \\\\end{array}$$
+AART:b = Y -(Cb + Cr) >> 2
+AART:r = Cr + b
+AART:g = Cb + b
+Figure: Description of the transformation of pixels from coded modified YCbCr color space to RGB color space (in case of exception). {#figureYcbcrRgbException}
+
+Background: At the time of this writing, in all known implementations of FFV1 bitstream, when `bits_per_raw_sample` was between 9 and 15 inclusive and `extra_plane` is 0, GBR Planes were used as BGR Planes during both encoding and decoding. In the meanwhile, 16-bit JPEG2000-RCT was implemented without this issue in one implementation and validated by one conformance checker. Methods to address this exception for the transform are under consideration for the next version of the FFV1 bitstream.
+
```
Comment (retokromer) [inline ffv1.md:415]: I’m happy this will be considered.

File: ffv1.md [hunk 4]
```diff
@@ -394,16 +389,40 @@ In JPEG2000-RCT, the coding order would be left to right and then top to bottom,
...
+AART:Cb = g - b
+AART:Cr = r - b
+AART:Y = b +(Cb + Cr) >> 2
+Figure: Description of the transformation of pixels from RGB color space to coded modified YCbCr color space (in case of exception). {#figureRgbYcbcrException}
+
+SVGI:!---
+SVGI:![svg](rgb4.svg "rgb 4")
+SVGI:!---
+SVGC:rgb4.svg=$$\\\\begin{array}{ccccccc}b & = & Y - (Cb + Cr)>>2 \\\\\\ r & = & Cr + b \\\\\\ g & = & Cb + b \\\\end{array}$$
+AART:b = Y -(Cb + Cr) >> 2
```
Comment (dericed) [inline ffv1.md:409]: add space after `-`

File: ffv1.md [hunk 5]
```diff
@@ -394,16 +389,40 @@ In JPEG2000-RCT, the coding order would be left to right and then top to bottom,
...
+
+If `bits_per_raw_sample` is between 9 and 15 inclusive and `extra_plane` is 0, the following formulae for reversible conversions between YCbCr and RGB MUST be used instead of the ones above:
+
+SVGI:!---
+SVGI:![svg](rgb3.svg "rgb 3")
+SVGI:!---
+SVGC:rgb3.svg=$$\\\\begin{array}{ccccccc}Cb & = & g - b \\\\\\ Cr & = & r - b \\\\\\ Y & = & b + (Cb + Cr)>>2 \\\\end{array}$$
+AART:Cb = g - b
+AART:Cr = r - b
+AART:Y = b +(Cb + Cr) >> 2
```
Comment (dericed) [inline ffv1.md:402]: Add space after `+`

File: ffv1.md [hunk 6]
```diff
@@ -710,6 +733,8 @@ Figure: A pseudo-code description of the read of a signed integer in Golomb Rice
 |0000 0000 0001 | 11    |
 |0000 0000 0000 | ESC   |
 
+`ESC` (meaning "Escape code") case is choosen when it is considered more efficient to store the whole number compared to storing the least significant bits of the number.
```
Comment (dericed) [inline ffv1.md:736]: `ESC` is already defined in Section 2.1. Is this a redefinition? If it is defined here, could it be removed from Section 2.1 so there are not competing places that define it.

**Review by dericed — COMMENTED** [review]
Thanks. I re-reviewed and added 3 comments.

**Comment by JeromeMartinez** [comment]
I added the missing spaces during the move of the section.
I changed the ESC commit to a move, from comments it looks like that ESC is not enough used for being worth to be at the beginning in the definitions.
@dericed please review again.

**Review by retokromer — APPROVED** [review]

**Review by dericed — APPROVED** [review]

**Comment by dericed** [comment]
thx, reviewed and tested, thanks for addressing all points.

**Comment by michaelni** [comment]
d1d3da1efb89efc96f5207cbf4ffcd37cb10aa06 adds a line thats removed in the next commit

**Comment by JeromeMartinez** [comment]
> d1d3da1 adds a line thats removed in the next commit

Fixed, please review again / merge.

**Review by dericed — APPROVED** [review]

**Pull request merged**